packages/ui-spinner/src/Spinner/props.ts

```diff
@@ -28,30 +28,20 @@ import type {
   ComponentStyle
 } from '@instructure/emotion'
 import type {
-  AsElementType,
   OtherHTMLAttributes,
   SpinnerTheme
 } from '@instructure/shared-types'
 import type { WithDeterministicIdProps } from '@instructure/ui-react-utils'
 import { Renderable } from '@instructure/shared-types'
 
 type SpinnerOwnProps = {
-  /**
-   * Render Spinner "as" another HTML element
-   */
-  as?: AsElementType
   /**
    * delay spinner rendering for a time (in ms). Used to prevent flickering in case of very fast load times
    */
   delay?: number
   /**
-   * provides a reference to the underlying html root element
-   */
-  elementRef?: (element: Element | null) => void
-  /**
-   * Valid values are `0`, `none`, `auto`, `xxx-small`, `xx-small`, `x-small`,
-   * `small`, `medium`, `large`, `x-large`, `xx-large`. Apply these values via
-   * familiar CSS-like shorthand. For example: `margin="small auto large"`.
+   * Valid values are from themes. See theme.semantics.spacing. Apply these values via
```

Comment:
This is theme specific, right? `theme.semantics.spacing` is not guaranteed to be this path, I'd rephrase it to "See "See `theme.semantics.spacing` in the Canvas theme for example"